guide/event-driven-and-pooling-consumer-service/event-driven-and-pooling-consumer.bal

```diff
@@ -0,0 +1,104 @@
+    url: "http://localhost:8080/maintenance"
+};
+documentation { Queue receiver endpoint for calibration messages. }
+endpoint mb:SimpleQueueReceiver queueReceiverCalibrate {
+    host: "localhost",
+    port: 5672,
+    queueName: "Calibration"
+};
+endpoint http:Client  CalibrateEP {
+    url: "http://localhost:8080/calibrate"
+};
+documentation { Service to receive messages to the new GeoActivities message queue. }
+service<mb:Consumer> geoListener bind queueReceiverGeoActivities {
+    documentation { Resource handler for new messages from queue. }
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        string messageText = check message.getTextMessageContent();
+        json requestPayload = messageText;
+        outRequest.setJsonPayload(untaint requestPayload);
+        var resp = GeoActivityEP->post("/", outRequest);
+        log:printInfo("Message recived from queue GeoActivities: " + messageText + "status:forwarded to /activity");
+    }
+}
+documentation { Service to receive messages to the SensorHealth message queue. }
+service<mb:Consumer> healthListener bind queueReceiverHealth {
+    documentation { Resource handler for new messages from queue. }
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        string messageText = check message.getTextMessageContent();
+        json requestPayload = messageText;
+        outRequest.setJsonPayload(untaint requestPayload);
+        var resp = HealthEP->post("/", outRequest);
+        log:printInfo("Message recived from queue SensorHealth: " + messageText + "status:forwarded to /health");
+    }
+}
+documentation { Service to receive messages to the Maintenance message queue. }
+service<mb:Consumer> maintenanceListener bind queueReceiverMaintenance {
+    documentation { Resource handler for new messages from queue. }
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        string messageText = check message.getTextMessageContent();
+        json requestPayload = messageText;
+        outRequest.setJsonPayload(untaint requestPayload);
+        var resp = MaintenanceEP->post("/", outRequest);
+        log:printInfo("Message recived from queue Maintenance: " + messageText + "status:forwarded to /maintenance");
+    }
+}
+documentation { Service to receive messages to the Calibration message queue. }
+service<mb:Consumer> calibrateListener bind queueReceiverCalibrate {
+    documentation { Resource handler for new messages from queue. }
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        string messageText = check message.getTextMessageContent();
+        json requestPayload = messageText;
+        outRequest.setJsonPayload(untaint requestPayload);
+        var resp = CalibrateEP->post("/", outRequest);
+        log:printInfo("Message recived from queue Calibration: " + messageText + "status:forwarded to /calibrate");
```

Comment:
recived -> received

Reply:
Fix the typo recived -> received in all places.